docs/testCaseReduction.md

```diff
@@ -0,0 +1,30 @@
+# Test Case Reduction
+SQLancer generates a large number of statements, but not all of them are relevant to the bug. To automatically reduce the test cases, two reducers were implemented: the statement reducer and the AST-based reducer.
+
+## Statement Reducer
+The statement reducer utilizes the delta-debugging technique to remove irrelevant statements. More details of delta-debugging could be found in this paper: [Simplifying and Isolating Failure-Inducing Input](https://www.cs.purdue.edu/homes/xyzhang/fall07/Papers/delta-debugging.pdf). 
+
+Using the statement reducer, SQLancer could effectively reduce the set of statements to a minimal subset that reproduces the bug. 
+
+## AST-Based Reducer
+The AST-based reducer can shorten a statement by applying AST level transformations, including removing unnecessary clauses, irrelevant elements in a list, simplify complicated expressions and etc. The transformations are implemented by JSQLParser, which supports the SQL standard as well as major DBMSs, 
```

Comment:
when mentioning JSQLParser here, perhaps you could add a link to the website and also explain that JSQLParser is (which is currently quite implicit)?